requirements-dev.txt

```diff
@@ -4,4 +4,5 @@ pytest-cov>=3.0.0
 pytest-watch>=4.2
 responses>=0.21.0
 pytest-mock>=3.8
-pytest-assume>=2.4.3
\ No newline at end of file
+pytest-assume>=2.4.3
+git+https://github.com/credo-ai/credoai_connect#develop
```

Comment:
https://github.com/credo-ai/credoai_lens/actions/runs/3913392041/jobs/6689228279#step:4:681 <- This log seems to confirm that develop branch of connect is the one being used for tests (Develop is v0.0.6 and Main is v0.0.5).

Agreed?